doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,75 @@
+---------------------------------------
+Finding and Downloading Data using Fido
+---------------------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified interface for seeking
+and downloading solar physics data irrespective of the underlining
+client or webservice through which the data is obtained, e.g. VSO,
+JSOC, etc.  It therefore supplies a single, easy and consistent way to
+obtain most forms of solar physics data.
+
+Import
+------
+
+SunPy's Fido module is in ``sunpy.net``.  It can be imported as follows::
+
+    >>> from sunpy.net import Fido, attrs as a
+
+Searching for Data Using Fido
+-----------------------------
+
+To search for data with Fido, your query needs at minimum a start time,
+an end time, and an instrument.  Enter these properties using SunPy's
+attrs module::
+
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('lyra'))
+
+This returns an `sunpy.net.fido_factory.UnifiedResponse` object
+containing information on the available online files which fit the
```

Comment:
would using "records" not files be better? There is not always a one-one correspondence of the two.